src/data/Providers.js

```diff
@@ -0,0 +1,11 @@
+import { Providers } from '../API';
+
+export const useMap = () => {
+  const query = Providers.useProviderQuery();
+  const map = Object.fromEntries(query.providers.map(l => [l.id, l.name]));
```

Comment:
`providers.reduce((acc, el) => {
 acc[el.id] = el.name;
 return acc;
}, {})`

more optimal, since we go through the array 1 time. But maybe yours looks more beautiful, if the data is not large, we can leave it as it is

Reply:
oops, sorry, it shows 2x slower in fact )

Reply:
Based on this benchmark reduce almost 2 times faster (and this is logical, since we go through the array 2 times). As I understand, the more executions per second, the more optimal the algorithm is.